build.sh

```diff
@@ -350,6 +350,18 @@ if [ $_autoselect_tests -eq 1 ]; then
     fi
 fi
 
+# If the `PB_SKIPTESTS` variable is set to 'true' then no tests should be built or run, even if explicitly specified
+if [ $PB_SKIPTESTS -eq "true" ]; then
```

Comment:
Same for unix. I have 0 experience with VSTS builds and unix hence am not sure about this. @mmitche will know though.

Reply:
This is incorrect for Unix.  It will thread through, but the casing will be the "correct" casing.  It will be PB_SkipTests.

Btw, I think it would make sense to just pass through some switch -SkipTests=$(PB_SkipTests) in your argument parsing, yeah?  That way it would be possible for devs to skip tests?

Reply:
According to [this](https://docs.microsoft.com/en-us/vsts/build-release/concepts/definitions/build/variables?tabs=batch) the variable will be upper-cased.